tests/2fa.ts

```diff
@@ -1,15 +1,54 @@
 import speakeasy from "speakeasy";
```

Comment:
Maybe you could provide a description of what this test file is testing for?

Reply:
This is not a test file, this is a utility file